### PR TITLE
Force Patroni and Redis to reapply deployment configs every time

### PR DESCRIPTION
**Pull request by jujaga**
# Description

If this is not done, existing patroni and redis instances will not be updated and redeployed.

NSPs never came into effect as it was skipped.

## Types of changes

Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the OpenAPI 3.0 `v*.api-spec.yaml` documentation (if appropriate)
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-9f0fbe-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-email-service-pr-36

File: Jenkinsfile [hunk 1]
```diff
@@ -276,28 +276,24 @@ def deployStage(String stageEnv, String projectEnv, String hostRouteEnv) {
...
-              def dcPatroniTemplate = openshift.process('-f',
-                'openshift/patroni.dc.yaml',
-                "INSTANCE=${JOB_NAME}"
-              )
-
-              echo "Applying Patroni StatefulSet..."
-              def dcPatroni = openshift.apply(dcPatroniTemplate).narrow('statefulset')
-              dcPatroni.rollout().status('--watch=true')
-            }
+            echo "Processing Patroni StatefulSet.."
```
Comment (loneil) [inline Jenkinsfile]: missing a dot

File: Jenkinsfile [hunk 2]
```diff
@@ -318,21 +314,17 @@ def deployStage(String stageEnv, String projectEnv, String hostRouteEnv) {
...
-                "REPO_NAME=${REPO_NAME}",
-                "JOB_NAME=${JOB_NAME}",
-                "APP_NAME=${APP_NAME}"
-              )
-
-              echo "Applying Redis Deployment..."
-              def dcRedis = openshift.apply(dcRedisTemplate).narrow('dc')
-              dcRedis.rollout().status('--watch=true')
-            }
+            echo "Processing DeploymentConfig Redis.."
```
Comment (loneil) [inline Jenkinsfile]: dot

File: Jenkinsfile.cicd [hunk 3]
```diff
@@ -255,21 +255,17 @@ def deployStage(String stageEnv, String projectEnv, String hostRouteEnv) {
...
-                "APP_NAME=${APP_NAME}",
-                "INSTANCE=${JOB_NAME}",
-                "NAMESPACE=${projectEnv}"
-              )
-
-              echo "Applying Patroni StatefulSet..."
-              def dcPatroni = openshift.apply(dcPatroniTemplate).narrow('statefulset')
-              dcPatroni.rollout().status('--watch=true')
-            }
+            echo "Processing Patroni StatefulSet.."
```
Comment (loneil) [inline Jenkinsfile.cicd]: dot

File: Jenkinsfile.cicd [hunk 4]
```diff
@@ -290,21 +286,17 @@ def deployStage(String stageEnv, String projectEnv, String hostRouteEnv) {
...
-                "REPO_NAME=${REPO_NAME}",
-                "JOB_NAME=${JOB_NAME}",
-                "APP_NAME=${APP_NAME}"
-              )
-
-              echo "Applying Redis Deployment..."
-              def dcRedis = openshift.apply(dcRedisTemplate).narrow('dc')
-              dcRedis.rollout().status('--watch=true')
-            }
+            echo "Processing DeploymentConfig Redis.."
```
Comment (loneil) [inline Jenkinsfile.cicd]: dot

**Pull request merged**